content/03.methods.md

```diff
@@ -49,6 +49,8 @@ For each target gene, in order to ensure that the training dataset was reasonabl
 After applying these filters, the number of valid cancer types remaining for each gene varied based on the set of samples used: more data types resulted in fewer shared samples, and fewer samples generally meant fewer valid cancer types.
 In some cases, this resulted in genes with no valid cancer types, which we dropped from the analysis.
 Out of the 511 genes from the "merged" cancer gene set described in the previous section, for the analysis using {expression, mutation} data we retained 268 target genes, for the {expression, mutation, 27k methylation, 450k methylation} analysis we retained 272 genes, and for the analysis using all data types we retained 217 genes.
+We also tried an alternate approach of including/excluding entire genes using our filters rather than individual cancer types: that is, we trained a classifier for each gene where all cancer types combined had at least 5% mutated samples and at least 100 total mutated samples.
+Results are shown in Supplementary Figure {@fig:all_volcano_filter}.
```

Comment:
We might get yelled at for this. A reference to supplementary results should really go within results somewhere. Probably needs a 1-liner somewhere in the results.